baybe/surrogates/gaussian_process/presets/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Gaussian process surrogate presets."""
+
+from baybe.surrogates.gaussian_process.presets.core import (
+    GaussianProcessPreset,
+    make_gp_from_preset,
+)
+
+__all__ = [
+    "make_gp_from_preset",
+    "GaussianProcessPreset",
```

Comment:
I assume you must already have thoughts about other settings that could be changed, but for now we only adjust the priors, why not call it `PriorPreset`

but if you have immediate other properties that would be set/changed I guess its better to keep it this way

Reply:
Yes, in fact, we [ARE already setting](https://github.com/emdgroup/baybe/pull/235#discussion_r1601028300) the likelihood model as well, though it's not yet controllable for the user. Therefore, I think it's better to keep it the way it is. But let me know how you prefer to proceed with the likelihood stuff, preferably by answering in the other thread 🦤  If you agree, please resolve